File: classes/python/MyDB.py
# Responsible for connecting to database

import sqlite3

class MyDB:
    conn = sqlite3.connect('courses.db')
   
    def connect(self):
        return self.conn


# TEST:
# def main():
#   db = MyDB()
#   conn = db.connect()
#   c = conn.cursor()
#   c.execute('SELECT * FROM Courses')
#   print c.fetchall()

# if __name__ == "__main__":
#     main()      

File: classes/php/MyDB.php
<?php
/**
* Responsible for scheduling logic, finding free times / users
*/

class MyDB extends SQLite3
{
    function __construct()
    {
        $this->open('courses.db');
    }
}




// EXAMPLE USAGE:

// $db = new MyDB();
		
// 		foreach ($users as $user){
// 			$sel = "SELECT p.firstName, c.days, c.startDate, c.endDate, c.startTime, c.endTime   FROM Courses c
//           		INNER JOIN CoursesProfs cp ON cp.cid = c.prof
//           		INNER JOIN Profs p ON p.email = cp.pid
//           		WHERE p.email = '".$user->email()."';";


// 			$result = $db->query($sel);

?>